hapistrano.cabal

```diff
@@ -52,13 +52,13 @@ library
                      , System.Hapistrano.Commands.Internal
                      , System.Hapistrano.Maintenance
   build-depends:       aeson              >= 2.0 && < 3.0
-                     , ansi-terminal      >= 0.9 && < 0.12
+                     , ansi-terminal      >= 0.9 && < 2.0
```

Comment:
I am keeping the original lower bound for all the dependencies where I bumped the upper bound to avoid introducing some breaking changes.